ngraph/python/tests/test_ngraph/util.py

```diff
@@ -45,8 +46,19 @@ def run_op_node(input_data, op_fun, *args):
     comp_args = []
     op_fun_args = []
     comp_inputs = []
-    for data in input_data:
-        op_fun_args.append(ng.constant(data, _get_numpy_dtype(data)))
+
+    for idx, data in enumerate(input_data):
+        node = None
+        if np.isscalar(data):
+            node = ng.parameter([], name=ascii_uppercase[idx], dtype=_get_numpy_dtype(data))
+        elif isinstance(data, (list)):
```

Comment:
I think we should be able to remove this block of code and fix the tests to use a NumPy array in all cases.

Reply:
Done